proxy_agent/src/proxy/proxy_summary.rs

```diff
@@ -6,6 +6,7 @@ use serde_derive::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize)]
 #[allow(non_snake_case)]
 pub struct ProxySummary {
+    pub id: u128,
```

Comment:
Is there any reason that the id for the Proxy Summary is so wide?

Reply:
yeah, we need id when log the proxy summary, we could use this id to lookup the details log.